discoverd/main.go

```diff
@@ -82,6 +85,58 @@ func (m *Main) Run(args ...string) error {
 		opt.Peers = []string{advertiseAddr}
 	}
 
+	// Create a slice of peers with their HTTP address set instead.
+	httpPeers, err := SetPortSlice(opt.Peers, opt.HTTPAddr)
+	if err != nil {
+		return fmt.Errorf("set port slice: %s", err)
+	}
+
+	// if there is a discoverd process already running on this
+	// address perform a deployment by starting a proxy DNS server
+	// and shutting down the old discoverd job
+	var deploy *dd.Deployment
+	var shutdownInfo dt.ShutdownInfo
+
+	target := fmt.Sprintf("http://%s:1111", opt.Host)
+	m.logger.Println("Trying to connect to:", target)
```

Comment:
I would probably say something like the following to make it clearer in the logs what is happening:

```
checking for existing discoverd process listening on xxx:1111
```